src/main/kotlin/carRace/domain/Car.kt

```diff
@@ -1,6 +1,11 @@
-package carRace
+package carRace.domain
+
+import java.lang.IllegalArgumentException
 
 data class Car(val driverName: String, private var _traveled: Int, val carMovingStrategy: CarMovingStrategy = RandomMovingStrategy()) {
+    init {
+        require(driverName.length <= 5) { throw IllegalArgumentException("Invalid driver name length $driverName") }
```

Comment:
```suggestion
        require(driverName.length <= 5) { "Invalid driver name length $driverName" }
```
이렇게만 작성해줘도 됩니다! require문 내에 예외 처리 로직이 포함되어있어요.